pkg/engine/variables/operator/in.go

```diff
@@ -0,0 +1,100 @@
+package operator
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/go-logr/logr"
+	"github.com/nirmata/kyverno/pkg/engine/context"
+)
+
+//NewInHandler returns handler to manage In operations
+func NewInHandler(log logr.Logger, ctx context.EvalInterface, subHandler VariableSubstitutionHandler) OperatorHandler {
+	return InHandler{
+		ctx:        ctx,
+		subHandler: subHandler,
+		log:        log,
+	}
+}
+
+//InHandler provides implementation to handle NotIn oerator
+type InHandler struct {
+	ctx        context.EvalInterface
+	subHandler VariableSubstitutionHandler
+	log        logr.Logger
+}
+
+//Evaluate evaluates expression with In Operator
+func (in InHandler) Evaluate(key, value interface{}) bool {
+	var err error
+	//TODO: decouple variables from evaluation
```

Comment:
what does this mean? if its a carry-over please remove

Reply:
Yes, it is carry-over. I have removed it.